app/presenters/result_set_presenter.rb

```diff
@@ -16,6 +16,7 @@ def initialize(finder, filter_params, view_context)
     @total = finder.results.total
     @filter_params = filter_params
     @view_context = view_context
+    @top_result_finder_whitelist = ["/find-eu-exit-guidance-business"]
```

Comment:
You could use a global variable here as the value never changes.

Reply:
We really shouldn't pepper finder-frontend with too many paths like this (usually the same eu-exit finder path).
There's a method on finder which (I added here https://github.com/alphagov/finder-frontend/blob/master/app/presenters/finder_presenter.rb#L345) which also contains this path. As @leenagupte says we should define this as a constant once somewhere where it can referenced easily in many methods/places.

Reply:
I agree with @steventux. Also FinderPresenter is being passed to ResultSetPresenter as `finder` so if we make the `eu_exit_finder` method public, ResultSetPresenter will be able to access it:

https://github.com/alphagov/finder-frontend/blob/master/app/controllers/finders_controller.rb#L50